### PR TITLE
Currency: Added build-script and build_info.json for eureka

### PR DESCRIPTION
**Pull request by pooja0805**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Have you validated script on UBI 9 container
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: e/eureka/eureka_ubi_9.3.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Install dependencies.
+yum install -y yum-utils git wget tar java-11-openjdk-devel
+
+# Clone hibernate-commons-annotations repo
+git clone $PACKAGE_URL
+cd  $PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+
+if ! ./gradlew clean build -x test; then
+    echo "Build Fails!"
```
Comment (nikhil-kalbande) [inline e/eureka/eureka_ubi_9.3.sh:34]: @pooja0805 We are approving Pr's with format. I will say let's get complete echo messages like other PR's.. e.g

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

File: e/eureka/eureka_ubi_9.3.sh (hunk 1 above)
Comment (pooja0805) [reply e/eureka/eureka_ubi_9.3.sh:34]: Sure. I will update accordingly

File: d/DefinitelyTyped/build_info.json [hunk 2]
```diff
@@ -1,12 +1,15 @@
 {
-  "maintainer": "shubham-garud",
+  "maintainer": "AbhiIBM",
   "package_name": "DefinitelyTyped",
   "github_url": "https://github.com/DefinitelyTyped/DefinitelyTyped",
-  "package_dir": "d/DefinitelyTyped/",
+  "package_dir": "d/definitelytyped/",
```
[the remark this answers is not in the document]
Comment (pooja0805) [reply d/DefinitelyTyped/build_info.json:5]: This change is not required. Let me remove this change.

**Comment by pooja0805** [comment]
@nikhil-kalbande Updated the echo commands according to the mentioned format.

**Pull request closed without merging**